core-web/nx.json

```diff
@@ -106,6 +106,16 @@
         "@nx/eslint:lint": {
             "inputs": ["default", "{workspaceRoot}/.eslintrc.json"],
             "cache": true
+        },
+        "@nx/rollup:rollup": {
+            "cache": true,
+            "dependsOn": ["^build"],
+            "inputs": ["production", "^production"]
+        },
```

Comment:
Do we need this?

Reply:
I leave it because of the other libs. But I can delete it.

Reply:
Deleted.